docs/_docs/troubleshooting.md

```diff
@@ -1,59 +0,0 @@
----
-layout: article
-title: Troubleshooting
-category: admin
----
```

Comment:
Seems like this file vanished from the new documentation.

Reply:
Yes, I've deleted the file